packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -1013,3 +1013,40 @@ input:-webkit-autofill {
   background-color: #f5f6fb;
   height: 34px;
 }
+
+.NotifiExpiredTokenCard__container {
+  background-color: var(--notifi-card-background);
```

Comment:
do we need a background color? won't the container have it?

Reply:
removed!